cyclonedx_py/parser/pipenv.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, pipenv_contents: str) -> None:
                     type='pypi', name=package_name, version=str(package_data.get('version') or 'unknown').lstrip('=')
                 )
             )
-            if package_data.get('index') == 'pypi' and isinstance(package_data.get('hashes'), list):
```

Comment:
i do not see a reason why the condition `package_data.get('index') == 'pypi'` was needed.
this is due to my lack of knowledge of pipenv files.

do you remember why this was needed, @madpah  ? 
or is there  some tech spec/docs to the format? 

I wonder if we gave regression tests or integration tests, that will fail with this implementation change.

Reply:
and there i see lots of tests failing. see https://github.com/CycloneDX/cyclonedx-python/actions/runs/2185186482 

please run the tests locally and have them fixed, @sleightsec 
see https://github.com/CycloneDX/cyclonedx-python/blob/master/CONTRIBUTING.md

Reply:
tldr: Fine to remove @jkowalleck , @sleightsec.

I think removing the condition `package_data.get('index') == 'pypi'` is fine - it was added to shield against spec changes (Pipenv specification remains under development!), but also because the only valid examples I had at the time had this included for every dependency.